### ISSUE TITLE
Trying to implement a GUI but don't want the tabs.

### ISSUE THREAD

**Issue by The-Code-Monkey** [TechReborn/RebornCore#160]
Hi,

I'm implementing a GUI but using your GUI code, is there a way for me to disable the tabs on the left as they will do things that we don't want the GUI to be able to do?

Also would be nice if the energy bar could be changed in width.

**Comment by drcrazy** [comment]
You can check gui for solar panels, IMHO. They do not have item\fluid i\o config tabs

**Comment by drcrazy** [comment]
Also as far as I can remember these tabs appearance depends on blockentity. So check solar panels blockentity also.

**Comment by The-Code-Monkey** [comment]
Ah ok I'm trying to basically make a battery that can charge via a slot in the gui

**Comment by jusvit** [comment]
> Ah ok I'm trying to basically make a battery that can charge via a slot in the gui

	@Override
	public boolean canBeUpgraded() {
		return false;
	}

	@Override
	public boolean hasSlotConfig() {
		return false;
	}

I believe those are the methods you have to override, however I was under the impression to were off by default but I guess not.

**Comment by The-Code-Monkey** [comment]
Cheers they look like what I expected cheers

**Comment by The-Code-Monkey** [comment]
Yeah @justinvvitale they got rid of the slotConfig but i still cant get rid of the redstone config and there doesnt seem to be a similar boolean for that one.
